ament_cmake_clang_format/cmake/ament_clang_format.cmake

```diff
@@ -15,15 +15,21 @@
 #
 # Add a test to check the code for compliance with clang_format.
 #
+# The configuration file for clang-format to use can be either set by
+# the function parameter 'CONFIG_PATH' or by a global variabled called
```

Comment:
> the function parameter ...

Proposing: the argument ...

> 'CONFIG_PATH'

Should be: 'CONFIG_FILE'

> variabled

Spelling: variable

> called

Proposing: named

Reply:
Thanks for pointing these out